reports/project-cycle-time.ts

```diff
@@ -84,7 +84,12 @@ export function renderMarkdown(projData: ProjectData, processedData: any): strin
     let ctRow = <CycleTimeRow>{};
     ctRow.labels = `\`${cardType}\``;
     ctRow.count = stageData.count;
-    ctRow.cycleTimeInDays = ` ${stageData.cycletime.toFixed(2)} ${stageData.flag ? ":triangular_flag_on_post:": ""}`;
+    if (stageData.cycletime > 0) {
+      ctRow.cycleTimeInDays = ` ${stageData.cycletime.toFixed(2)} ${stageData.flag ? ":triangular_flag_on_post:": ""}`;
```

Comment:
Maybe this since 0 is "falsey"?

```javascript 
`  ${stageData.cycletime.toFixed(2)} || "" ...`
```

Reply:
hmmm - trying

Reply:
Actually, since 0.00 doesn't seem to be falsey, this works.

What do you think?

```javascript
` ${stageData.cycletime ? stageData.cycletime.toFixed(2):""}`
```